packages/tldraw/src/lib/ui/components/primitives/TldrawUiSlider.tsx

```diff
@@ -11,13 +10,20 @@ export interface TLUiSliderProps {
 	label: string
 	title: string
 	onValueChange(value: number): void
+	onHistoryMark(id: string): void
```

Comment:
Same question here. `onHistoryMark` is so editor specific, do we see any uses here outside of editor. If nothing else then I'd at least rename it.

Reply:
maybe? If there's an app level undo/redo system? we could do the useMaybeEditor instead though